sound/hybrid_font.h

```diff
@@ -553,6 +559,25 @@ class HybridFont : public SaberBase, public Looper {
 	return;
       case EFFECT_PREON: SB_Preon(location); return;
       case EFFECT_POSTOFF: SB_Postoff(); return;
+      case EFFECT_ACCENT_SWING:
+        if (guess_monophonic_) {
```

Comment:
I think this misses the CFX case, where SFX_humm is being used, and SFX_swing is being played, but in polyphonic mode.

Maybe this?

if (SFX_swng) {
   swing_player_ = PlayPolyphonic(&SFX_swng);
} else if (!guess_monophonic_) {
   swing_player_ = PlayPolyphonic(&SFX_swing);
} else {
   PlayMonophonic(&SFX_swing, &SFX_hum);
}

This prefers SFX_swng if available, which isn't exactly how it worked before, but it seems reasonable to me.

Reply:
ok I made it that.
Monophonic fonts are going to be the death of me lol.